JShellAPI/src/main/java/org/togetherjava/jshellapi/service/DockerService.java

```diff
@@ -59,22 +62,30 @@ private void cleanupLeftovers(UUID currentId) {
 
     public String spawnContainer(long maxMemoryMegs, long cpus, @Nullable String cpuSetCpus,
             String name, Duration evalTimeout, long sysoutLimit) throws InterruptedException {
-        String imageName = "togetherjava.org:5001/togetherjava/jshellwrapper";
+
+        String[] imageNameParts = this.jshellWrapperImageName.split(":master");
+
+        if (imageNameParts.length != 1) {
+            throw new IllegalArgumentException("invalid jshellWrapper image name");
+        }
```

Comment:
This should probably be checked before, in the config constructor

Reply:
It is in the constructor !

Reply:
I don't see a reason to do that the parts aren't needed in the constructor!

Reply:
But it isn't the responsibility of this class to check this